src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -480,23 +482,24 @@ public void should_return_a_200_with_search_results_based_on_jurisdiction_and_lo
         final String taskId = taskVariables.getTaskId();
 
         SearchTaskRequest searchTaskRequest = new SearchTaskRequest(asList(
-            new SearchParameter(JURISDICTION, SearchOperator.IN, singletonList("IA")),
-            new SearchParameter(LOCATION, SearchOperator.IN, singletonList("765324"))
+            new SearchParameterList(JURISDICTION, SearchOperator.IN, singletonList("IA")),
+            new SearchParameterList(LOCATION, SearchOperator.IN, singletonList("765324"))
         ));
 
         common.setupCFTOrganisationalRoleAssignment(headers);
 
         common.insertTaskInCftTaskDb(taskVariables, "followUpOverdueReasonsForAppeal", headers);
 
         Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED + "?first_result=0&max_results=10",
+            ENDPOINT_BEING_TESTED,
             searchTaskRequest,
             headers
         );
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.size()", lessThanOrEqualTo(10))
+            .body("tasks.size()", lessThanOrEqualTo(50)) //Default max results
+            .body("tasks.id", hasItem(taskId))
```

Comment:
With regards to asserting tasks.id, there was an issue in AAT a while back where the task was not returned within the limit required. Any chance this task could be 50 + n and therefore not returned?

Reply:
If pagination is not specified, then default records returned are 50. So no chance of returning 50+n.